### ISSUE TITLE
TouchKit Lite Documentation

### ISSUE THREAD

**Issue by Sposito** [prime31/TouchKit#53]
There is any kind of TouchKitLite documentation, I can't find it anywhere, it is kinda of different from standard touch and I can't find myself on code (even it beeing so small).

**Comment by prime31** [comment]
TouchKitLite doesn't actually do anything besides gathering touches and providing an API that works in the editor as well. There isn't really much to document, just read through the code.

**Comment by Sposito** [comment]
Yeah, eventually I got it, but the way it receives the Action is kinda different from the main Touch kit. Thanks anyway! =D
